.github/workflows/image-build-debian-source.yml

```diff
@@ -0,0 +1,93 @@
+##
+## Build the Docker Debian image from source
+##
+
+name: Build Linux Source Image
+
+env:
+  REPO_OWNER: stacks-network
+
+# Only run when:
+#   - workflow is manually triggered
+#   - manually triggered via the ci.yml workflow
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    inputs:
+      docker_platforms:
+        required: true
+        description: "Arch to buid images"
+        type: string
+        default: linux/amd64
+      linux_version:
+        required: true
+        description: "Linux image to build"
+        type: string
+        default: debian
+      build_type:
+        required: true
+        description: Build type (source/binary)
+        type: string
+        default: source
+    secrets:
+      DOCKERHUB_USERNAME:
+        required: true
+      DOCKERHUB_PASSWORD:
+        required: true
+
+jobs:
+  image:
+    name: Build Image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        id: git_checkout
+        uses: actions/checkout@v3
+      - name: Set Vars
+        id: set_vars
+        run: |
+          echo "GITHUB_SHA_SHORT=${GITHUB_SHA::7}" >> $GITHUB_ENV
+          echo "GITHUB_REF_SHORT=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Set up QEMU
+        id: docker_qemu
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        id: docker_buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Extract branch name
+        id: extract_branch
+        if: ${{ github.event_name != 'pull_request' }}
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/})" >> $GITHUB_ENV
+      - name: Extract branch name (PR)
+        id: extract_branch_pr
+        if: ${{ github.event_name == 'pull_request' }}
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_HEAD_REF})" >> $GITHUB_ENV
+      - name: Docker Metadata
+        id: docker_metadata
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ${{ env.REPO_OWNER }}/${{ github.event.repository.name }}
```

Comment:
I think this can be reduced to `${{ github.repository }}`, and the `REPO_OWNER` env var removed.

Reply:
same as for the `image-build-alpine-binary.yml` comment, will test and come back

Reply:
https://github.com/wileyj/stacks-blockchain/commit/d8ce51e8d9a4990728bc43acf5e48c80a8079103